src/makers/heltec/index.ts

```diff
@@ -0,0 +1,9 @@
+import antennas from './antennas'
+import hotspots from './hotspots'
+
+export default {
+  antennas,
+  hotspots,
+  id: 99999, // TODO: Get maker id from onboarding server
```

Comment:
pending official maker id

Reply:
This part will be updated when our maker account is created. 😃

Reply:
Already updated maker ID, please review again. 😄